src/components/BadgerBadge/stories.js

```diff
@@ -316,9 +316,12 @@ storiesOf('BadgerBadge', module)
 			<BadgerBadge
 				paymentRequestUrl={text(
 					'Invoice URL',
-					'https://yourInvoiceUrlHere.com/String'
+					//'https://yourInvoiceUrlHere.com/String'
+					'https://pay.bitcoin.com/i/9S21uXfxguax3RtewXmWch'
 				)}
 				showQR={boolean('showQR', true)}
+				showBorder={boolean('Toggle Border', true)}
+				showBrand={boolean('Badger info', true)}
```

Comment:
Let's remove these 2 props, as the knobs should be minimal for the purpose of the example shown.

Reply:
Done